docs/string.md

```diff
@@ -16,6 +16,13 @@
 * [`encodeURIComponent`](#encodeuricomponent) 
 * [`repeat`](#repeat) 
 * [`truncate`](#truncate) 
+* [`slugfy`](#slugfy) 
+* [`endsWith`](#endsWith) 
+* [`stripTags`](#stripTags) 
+* [`latinize`](#latinize)
+* [`startWith`](#startWith)
```

Comment:
The markdown link after the # character does not take uppercase character. Write it all in lowercase.